starter-core/src/main/java/io/micronaut/starter/feature/spring/SpringTx.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2020 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.starter.feature.spring;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+import io.micronaut.starter.application.ApplicationType;
+import io.micronaut.starter.feature.Category;
+import io.micronaut.starter.feature.Feature;
+
+import javax.inject.Singleton;
+
+@Singleton
+public class SpringTx implements Feature {
```

Comment:
Users should be using the Spring annotations to define transactions when this feature is selected that means this feature also needs to apply the Spring feature when selected since `micronaut-spring-annotation` needs to be on the annotation processor classpath. 

See https://github.com/micronaut-projects/micronaut-spring/issues/73